src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +16,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+        if (user != null && user.getPassword().equals(password)) {
+            return true;
+        } else {
+            return false;
+        }
+
```

Comment:
Good, but can we simplify it only to one line of code?

Also, don't forget about the empty line.